pkg/collector/corechecks/cluster/helm/release.go

```diff
@@ -80,7 +80,10 @@ func decodeRelease(data string) (*release, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	if len(b) < 4 {
+		// Avoid panic if b[0:3] cannot be accessed
+		return nil, fmt.Errorf("The byte array is too short (expected at least 4 characters, got %s instead): it cannot contain a Helm release", fmt.Sprint(len(b)))
```

Comment:
Resolving : as discussed internally, the error message will not surface to the user directly as it would be encapsulated into a Debug log giving more context : `Error while decoding Helm release`